src/box/lua/info.c

```diff
@@ -610,7 +610,7 @@ lbox_info_init_static_values(struct lua_State *L)
 {
 	/* tarantool version */
 	lua_pushstring(L, "version");
-	lua_pushstring(L, tarantool_version());
+	lua_pushfstring(L, "%s %s", tarantool_package(), tarantool_version());
```

Comment:
I'm afraid external modules can parse this string and will fails so. Why not to add a separate field?

Reply:
In earlier version it was separated. In first review solution was forced to be consistent with cmd tarantool --version